installer_scripts/install-yb-voyager

```diff
@@ -299,6 +304,20 @@ update_bashrc() {
 	done
 }
 
+insert_yb_session_vars_file() {
+	sudo touch /etc/ybSessionVariables.sql
+	sudo sh -c 'printf "SET client_encoding TO '\''UTF8'\'';\nSET session_replication_role TO replica;" > /etc/ybSessionVariables.sql'
+}
```

Comment:
Create a `yb-voyager/files/ybSessionVariables.sql` file and commit it in the git repo. Then you should be able to copy it just like ora2pg.conf file.

The file should be placed at path `/etc/yb-voyager/ybSessionVariables.sql`.

If the file already exists, then this `create_yb_session_vars_file()` function should be a no-op. Else it should download the file from the repo.

Also add the following two lines in the file:
```
SET yb_disable_transactional_writes to false;
SET yb_enable_upsert_mode to true;
```